Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -70,8 +79,8 @@ public final class ExploreView: UIView {
         return refreshControl
     }()
 
-    private lazy var collectionViewDataSource: UICollectionViewDiffableDataSource<ExploreSectionViewModel, Item> = {
-        let dataSource = UICollectionViewDiffableDataSource<ExploreSectionViewModel, Item>(
+    private func configureCollectionViewDataSource() -> DataSource {
+        let ds = DataSource(
```

Comment:
Can we change the variable name to `dataSource`?

Reply:
yeah, that will look better